docs/source/pillars/information_governance.md

```diff
@@ -188,11 +188,8 @@ _The ability of the TRE organisation to ensure that people with access to data a
 * - Statement
   - Guidance
   - Importance
-* - You have relevant training for all roles within the TRE organisation, and the ability to deliver this training
-  - This may include: Cyber security training, GDPR training, and higher level training for system operators
-  - Mandatory
-* - All TRE organisation members have completed relevant training within the last 12 months
```

Comment:
I don't think naming a specific length of time here is helpful as this would presumably vary depending on the training

Reply:
I was creating an edit for that part too - here's what I came up with as text:

* - All TRE organisation members have completed relevant training within an appropriate time-frame
  - We recommend that training be completed within the 12 months before access is given, and to maintain access training should be refreshed regularly such that no longer than 12 months has elapsed since completing training. The precise period may differ according to organisational policy and legal requirements

but seeing your edit, I agree that I'm not sure we need it at all - it's kind of covered by the final point in that table about the monitoring process.

Reply:
Looks like @jemrobinson has also edited this in #150 - I think his is more comprehensive so I will probably close this

Reply:
Sorry for the overlap - I didn't realise there was a PR open for this @edwardchalstrey1 !

Reply:
Also my fault for delaying my review of yours!